### PR TITLE
Workdir fix

### PR DESCRIPTION
**Pull request by garlicnation**

### CODE REVIEW

File: bin/wcs.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ var cli = cliArgs([
   {
     name: "workdir",
     type: String,
-    alias: "d",
+    alias: "w",
```
Comment (rictic) [inline bin/wcs.js]: Need to update README.md with this change

File: wcs.js [hunk 2]
```diff
@@ -82,12 +82,15 @@ WebComponentShards.prototype = {
   _synthesizeImport: function _synthesizeImport() {
     return this._getCommonDeps().then(function(commonDeps) {
       var synthetic = '';
+      var workdir = url.resolve(this.root, this.workdir);
+      var importPath = url.resolve(workdir, this.shared_import);
+      var walk = path.relative(importPath, workdir);
```
Comment (rictic) [inline wcs.js]: `walk` isn't a very clear variable name. workdirFromImport?

Comment (rictic) [inline wcs.js]: The variable names here in general are kinda tricky to read, and a comment at the top of what we're doing would go a long way. What do you think of this?

```javascript
      /** Generate the file of shared imports. */
      var output = '';
      var workdirPath = url.resolve(this.root, this.workdir);
      var outputPath = url.resolve(workdirPath, this.shared_import);
      var baseUrl = path.relative(outputPath, workdirPath);
      for (var dep in commonDeps) {
        output += '<link rel="import" href="' + baseUrl + '/' + commonDeps[dep] + '">\n';
      }
      var outDir = path.dirname(outputPath);
      mkdirp.sync(outDir);
      var fd = fs.openSync(outputPath, 'w');
      fs.writeSync(fd, output);
      return commonDeps;
```

**Comment by garlicnation** [comment]
@rictic Updated per your comments, PTAL.

**Comment by rictic** [comment]
Can you double check? I don't see your changes

**Comment by rictic** [comment]
LGTM

**Pull request merged**